examples/touch_n.py

```diff
@@ -0,0 +1,37 @@
+import luigi
```

Comment:
* Can you add a how-to-run-header for this file?
 * Is `touch_n ` a good name fir this fie? I'm thinking maybe naming it something to clarify this illustrates lazy completion checks.

Reply:
- I'll add a header. 
- This is not just for lazy completion checks. I use this to simulate long running tasks with (simple) outputs. Nevertheless I agree it could use a better name.